src/components/map-viewer.jsx

```diff
@@ -251,7 +251,10 @@ const MapViewer = ({
             <Box sx={styles.mapAndTreeContainer}>
                 {/* Waiting for map geodata is unnecessary. The map has is proper loader implementation */}
                 {/* This WaitingLoader is placed here to block functionnalities, hiding under components with some opacity*/}
-                <WaitingLoader message={'LoadingRemoteData'} loading={!isNetworkModificationTreeModelUpToDate} />
+                <WaitingLoader
+                    message={'LoadingRemoteData'}
+                    loading={!isNetworkModificationTreeModelUpToDate || isRootNetworksProcessing}
```

Comment:
Not sur e want to activate this big `WaitingLoader` every time we add or delete a root network.
Maybe use `isRootNetworksProcessing ` in `renderRootNetworksList` in `src/components/graph/menus/root-network-node-editor.tsx` and disable components here or render a MUI skeleton if true.